wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,212 @@
-| Click on hit object | **Left-Click:** Select / Move hit object against timeline. **Right-Click:** Remove hit object. |
-
-### Top-Right (Beat Snap Divisor)
-
-![Beat Snap Divisor](img/BE_BSD.jpg "Beat Snap Divisor")
-
-**In layman's's term, adjusting given length of a coloured sewing thread (timing section) for an embroidery design space (measures, divided by the divisor to be called beats) on the cloth (song).**
-
-Hold `Alt` to trigger Distance Snap. The value can range from x0.1–x2.0.
-
-#### Divisor 1/1
-
-![Divisor 1/1](/wiki/shared/BSD_1_1b.jpg "Divisor 1/1")
-
-- Full Beats
-- White ticks only
-- Best used for Easy difficulty
-
-#### Divisor 1/2
-
-![Divisor 1/2](/wiki/shared/BSD_1_2.jpg "Divisor 1/2")
-
-- Half Beats
-- Adds red ticks
-
-#### Divisor 1/3
-
-![Divisor 1/3](/wiki/shared/BSD_1_3.jpg "Divisor 1/3")
-
-- Triplets*
-- Has two purple ticks between each white tick
-- Compound timing
-
-#### Divisor 1/4
+| `+`/`-` buttons | Increases/Decreases timeline zoom. |
+| Double white vertical lines | Current timestamp with respect to the hit objects timeline. |
+| Click on hit object | `Left click`: Select/Move hit object against timeline. `Right click`: Remove hit object. |
 
-![Divisor 1/4](/wiki/shared/BSD_1_4.jpg "Divisor 1/4")
+### Beat snap divisor
 
-- Quarter Beats
-- Adds blue ticks, in addition to the red ticks
-- Useful for general mapping
+![](img/beat-snap-divisor.jpg "Beat snap divisor")
 
-#### Divisor 1/6
-
-![Divisor 1/6](/wiki/shared/BSD_1_6.jpg "Divisor 1/6")
-
-- Double Triplets*
-- Has two purple ticks between each red tick
-- Compound timing
-
-#### Divisor 1/8
-
-![Divisor 1/8](/wiki/shared/BSD_1_8.jpg "Divisor 1/8")
-
-- Eighth Beats
-- Adds yellow ticks, in addition to the red and blue
-
-**Insert Break Time** button will start/stop "Break Time" in current timestamp and **x/y** is current mouse coordinate.
-
-### Centre-Left (Selector)
-
-![Hit Notes Selector](img/BE_HO.jpg "Hit Notes Selector")
-
-**Your construction toolkit, made for DIY (do it yourself).**
+| Name | Description |
+| :-- | :-- |
+| [Beat Snap Divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor) | Limits where hit objects can be placed so that they always fall on the beat. Moving the slider to the right increases the granularity of the hit objects. |
+| `Insert Break Time` | This button will start/stop [break](/wiki/Beatmap/Break) time in current timestamp. |
+| x/y | The current mouse coordinate. |
 
-**Please note that in osu!, default note hitsounds work differently than in osu!mania.**
+Hold `Alt` to trigger **Distance Spacing**. The value can range from 0.1x to 6.0x.
 
-- Adding clap on a notes will play `hitclap` + `hitnormal` sound on hit while in osu!mania: will only play `hitclap` sound on hit
+### Selector
 
-#### Selector
+![](img/tools.jpg "Hit objects selector")
 
-| Buttons / Header (Keyboard Shortcut) | Usage | Description |
+| Buttons (Keyboard shortcut) | Usage | Description |
```

Comment:
additionally, I think that the usage and description can be safely combined in this table and a few down below, as they cover roughly the same things
```suggestion
| Button (Keyboard shortcut) | Usage | Description |
```